sys/log/full/pkg.yml

```diff
@@ -30,20 +30,18 @@ pkg.deps:
     - "@apache-mynewt-core/sys/log/common"
     - "@apache-mynewt-core/util/cbmem"
 
-pkg.deps.'LOG_CLI && LOG_VERSION > 2':
-    - "@apache-mynewt-core/sys/log/util/log_cbor_reader"
-
 pkg.deps.LOG_FCB:
     - "@apache-mynewt-core/hw/hal"
     - "@apache-mynewt-core/fs/fcb"
 pkg.deps.LOG_FCB2:
     - "@apache-mynewt-core/fs/fcb2"
 pkg.deps.LOG_CLI:
+    - "@apache-mynewt-core/sys/log/util/log_cbor_reader"
```

Comment:
Was this addition intentional?

Reply:
yes, this is intentional. It was
```
pkg.deps.'LOG_CLI && LOG_VERSION > 2':
    - "@apache-mynewt-core/sys/log/util/log_cbor_reader"
```
which I removed above and combined it below.